backend/lib/index.js

```diff
@@ -17,6 +18,8 @@ function createApp() {
   app.use(cors());
   app.use(pinoExpress(pino));
   app.use(express.json());
+  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded());
```

Comment:
Can you please try to remove the body parser and see if still works?

Reply:
Without body-parser `req.body` is `{}` when it should be something like `{ email: test@gmail.com, firstName: Test }` in the use case for `/api/users/auth`. Do you have any suggestions to get this to work without body-parser?

Reply:
Right now without body-parser the response when I login/register is the following since `req.body` is `{}` 

`{
    "name": "First name field is required",
    "email": "Email is invalid"
}`

Reply:
also try use `express.json(), express.urlencoded()`

Reply:
It seems that it's because this endpoint uses a GET method, and to pass a body payload we need to use POST method

Reply:
Yes only POST or PUT usually have JSON body otherwise can use query params with GET
you can see in docs this is recommended to only add it on the specific routes where it's needed: https://expressjs.com/en/resources/middleware/body-parser.html

Reply:
`/api/users/auth` is a POST method